docs/mast/mast.rst

```diff
@@ -868,6 +868,97 @@ To access sector information at a particular location there is  `~astroquery.mas
                 tess-s0010-1-4     10      1   4
 
 
+Zcut
+====
+
+Zcut for MAST allows users to request cutouts from various Hubble deep field surveys. The cutouts are
+returned as full-frame image (FFI) cutouts. This tool can be accessed in Astroquery by using the Zcut
+class.
+
+
+Cutouts
+-------
+
+The ~astroquery.mast.ZcutClass.get_cutouts function takes a coordinate and cutout size (in pixels or
```

Comment:
References to functions need to be surrounded by backticks so they can be rendered as links. 
I.e. \`~astroquery.mast.ZcutClass.get_cutouts\`.
The tilde tells Sphinx to only show the function name (get_cutouts) rather than the full call (astroquery.mast.ZcutClass.get_cutouts).